index.html

```diff
@@ -6,45 +6,48 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Campos e Requisitos da Tela de Votação</title>
     <link rel="stylesheet" href="style.css">
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@1.0.2/css/bulma.min.css" media="print" onload="this.media='all'">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@1.0.2/css/bulma.min.css">
 </head>
 
 <body>
 
-    <section class="section">
-        <div class="columns is-centered">
-            <div class="column is-half box">
-                <h1 class="title has-text-centered has-text-primary is-size-2 has-text-weight-bold">
-                    O que você acha do novo design?
-                </h1>
-
-                <form action="" method="post">
-                    <fieldset>
-                        
-                        <div class="columns is-centered mt-4">
-                            <div class="control">
-                                <input type="radio" id="otimo" name="voto" value="Ótimo!" required>
-                                <label for="otimo">Ótimo!</label>
-                                <br>
-
-                                <input type="radio" id="bom" name="voto" value="Bom, mas pode melhorar">
-                                <label for="bom">Bom, mas pode melhorar</label>
-                                <br>
-
-                                <input type="radio" id="nao-gostei" name="voto" value="Não gostei">
-                                <label for="nao-gostei">Não gostei</label>
-                            </div>
+    <section class="section is-fullheight is-flex is-justify-content-center is-align-items-center ">
+
+        <div class="column is-half box  is-flex-direction-column ">
+            <h1 class="title has-text-centered has-text-primary is-size-2 has-text-weight-bold ">
+                O que você acha do novo design?
+            </h1>
+
+            <form action="" method="post" class="is-flex is-flex-direction-column is-align-items-center ">
```

Comment:
Remover o `action`, pois não tem nada dentro dele e remover o espacinho que ficou **depois** do `is-align-items-center`.

Reply:
check